streaming/src/main/scala/io/gearpump/streaming/executor/Executor.scala

```diff
@@ -179,11 +187,28 @@ class Executor(executorContext: ExecutorContext, userConf : UserConfig, launcher
 
     case query: QueryExecutorConfig =>
       sender ! ExecutorConfig(systemConfig)
+
+    case registered: TaskRegistered =>
+      LOG.debug(s"Register task ${registered.taskId} succeed")
+    case rejected: TaskRejected =>
+      LOG.info(s"Task ${rejected.taskId} is rejected by AppMaster, shutting down myself...")
+      tasks.get(rejected.taskId).foreach(_ ! PoisonPill)
+  }
+
+  def taskMsgHandler: Receive = {
+    case register: RegisterTask =>
+      sendMsgWithTimeOutCallBack(appMaster, register, registerTaskTimout, registerTaskTimeOut())
+  }
+
+  private def registerTaskTimeOut(): Unit = {
```

Comment:
better name the function to timeOutBack to differentiate it from the val `registerTaskTimout`